app/kumactl/data/install/k8s/control-plane/kuma-cp/app.yaml

```diff
@@ -63,6 +63,40 @@ data:
     #
     # see conf/kuma-cp.conf for available settings
 ---
+apiVersion: v1
```

Comment:
I know this was in the original code, but I don't think we need it. Default config covers those values.